Automattic/wp-calypso
# Design: Update plan icons.

## How to test

You should be able to find new plan icons in the following pages:
- Plans page in the signup flow
  - Demo screenshot: [desktop](https://cloudup.com/i9zbrRHD55T), [mobile](https://cloudup.com/i0j4PrHQZ4m)
- Manage Purchases page(/me/purchases)
  - Demo screenshot: https://cloudup.com/iisLn8838vA
  - Click on a purchased item then you will see a plan icon followed by plan name.
- Choose a site and select Plan menu:
  - There will be a new plan icon on the page if the site is on a paid plan.
  - Demo screenshot: https://cloudup.com/iXxZQY1ixQu

This PR is a part of #14863 which is going to be dropped soon.

cc @cristelrossignol @markryall @fditrapani
